app/models/project_creation.rb

```diff
@@ -0,0 +1,9 @@
+module ProjectCreator
+  def self.generate(contractor, tribe, params)
+    params.merge({
+      total: tribe.total,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.